css/avatar.css

```diff
@@ -0,0 +1,65 @@
+
+.neog__avatar {
+    border-radius: var(--neog-border-radius-full);
+    border: 1px solid var(--neog-white);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-weight: var(--neog-text-semibold);
+    color: var(--neog-white);
+}
+.neog__avatar img{
+    width: 100%;
+    height: 100%;
+    border-radius: var(--neog-border-radius-full);
+    object-fit: cover;
+    object-position: top;
+}
+
+.neog__avatar .neog__badge{
+    position: absolute;
+    top: -0.25rem;
+    right: -0.25rem;
+}
+
+.neog__avatar.xl{
+    width: 8rem;
+    height: 8rem;
+    font-size: 4rem;
+}
+
+.neog__avatar.lg{
+    width: 6rem;
+    height: 6rem;
+    font-size: 3rem;
+}
+
+.neog__avatar.md{
```

Comment:
instead .neog__avatar.md you can directly use .md

Reply:
UI looks good to me

Reply:
Thank you so much Arul ji for the review/comments. I thought, in a whole application perspective ".md" is a global class name and this class name can be used for any component for different responsive scenarios. So to make it personalized for avatar, just grouped the class names.